### ISSUE TITLE
When Active Learning Card is displayed and users take "None of the above." Bot is not responding "Thanks for the feedback."

### ISSUE THREAD

**Issue by eduardmartos** [microsoft/BotBuilder-Samples#2172]
## Sample information
1. Sample type: [\samples\]
2. Sample language: [dotnetcore]
3. Sample name: 49.qnamaker-all-features

## Describe the bug
When Active Learning Card is displayed and users take "**None of the above.**" 
 (**DefaultCardNoMatchText**), everything in the template suggests that the Bot's response should be  "**Thanks for the feedback.**" (**DefaultCardNoMatchResponse**), but instead of this, Bot is not responding nothing.

## To Reproduce
1. Create a new Bot using this template
2. Create a new QnAMaker KB and activate Active Learning on it
3. Test in Emulator with utterances that trigger Active Learning card
4. See that nothing is responded, but on Emulator we can see this:
Activity.type = "Thanks for the feedback."

## Expected behavior
Activity.type = "message"
Activity.text = "Thanks for the feedback."

## Additional context
I solved this issue changing this lines on **QnAMakerBaseDialog.cs** class and **GetQnAResponseOptionsAsync** method:
from:
```
var cardNoMatchResponse = new Activity(DefaultCardNoMatchResponse);
```
to:
```
var cardNoMatchResponse = (Activity)Activity.CreateMessageActivity();
cardNoMatchResponse.Text = DefaultCardNoMatchResponse;
```

[bug]

**Comment by anusharavindrar** [comment]
@eduardmartos I tested the sample and I am able to repro the issue you are getting above. 
The line `var cardNoMatchResponse = new Activity(DefaultCardNoMatchResponse)` will not have an activity type. Hence the bot is not responding with anything, although it does show up in the emulator logs. Instead of using the changes you suggested above, we  can make use of `var cardNoMatchResponse = (Activity)MessageFactory.Text(DefaultCardNoMatchResponse)` , and the bot will respond back with 'Thanks for the feedback.'

- Using the sample:

![GHqnaActiveL_LI](https://user-images.githubusercontent.com/48731427/73884064-f03af780-4819-11ea-96bb-930a031dd820.jpg)


- After making the following change:

`var cardNoMatchResponse = (Activity)MessageFactory.Text(DefaultCardNoMatchResponse)`

![GHqnaActiveL!](https://user-images.githubusercontent.com/48731427/73884131-1365a700-481a-11ea-828a-a1facef501c1.png)

I have filed a [PR] #2174  to fix the sample to work as expected with the above changes.
